gluten-core/src/main/scala/io/glutenproject/execution/FileSourceScanExecTransformer.scala

```diff
@@ -123,11 +123,18 @@ class FileSourceScanExecTransformer(@transient relation: HadoopFsRelation,
 
   override def doValidateInternal(): Boolean = {
     // Bucketing table has `bucketId` in filename, should apply this in backends
-    if (!bucketedScan) {
-      super.doValidateInternal()
-    } else {
-      false
+    if (bucketedScan) {
+      logWarning(s"Validation failed for ${this.getClass.toString} due to " +
```

Comment:
Please use `logValidateFailure` to track all the failed validation

Reply:
Changed to use `logValidateFailure`, thanks.